### ISSUE TITLE
General feedback across site

### ISSUE THREAD

**Issue by anissa-talantikite** [DataUSA/datausa-site#486]
* On the home page, when “graphic design” auto-populates in the text box, the drop down menu reads “6 digit course” or “four digit course” under the terms. I’m not sure what that means and why it’s relevant, or if it’s a bug.
* Also on the home page, it might be helpful to provide a bit more guidance to the user on how to start navigating from there. A few ideas that would be minor tweaks:
  * Put the words “Explore visualizations” above the word “Profiles” in a larger font, and then list “Profiles, Stories, Maps.. etc.”
  * Add a one-sentence summary of each item, as in “Stories – “read articles that dive deeper into specific data” 
* On the “About” page – it might be helpful to provide some marketing/press information that can be easily pulled by journalists who want to write about Data USA (which we could provide). 
  * You also might want to make more clear that Macro Connections is a part of MIT Media Lab
* When I went to Cart > clicked on a county > Options, I wasn’t able to choose from the 4 options (Share, Download, Data, API). Just showed a blank screen.
* For “Wages by Race & Ethnicity for Common Jobs” across all of the US you may want to change the “Race  or Ethnicity with Highest Average Salary” to Asian-American rather than Asian. The text reads “Asian citizens earn, on an average…” but I think you mean Asian-Americans.
* For “Medicare over Time”, “Conditions/Patient Cohorts”, “Women’s Health” and “Diabetic Care” across all of the US, there’s no data available. Not sure if that’s a bug or if it’s data that’s being uploaded now

**Comment by rexley** [comment]
Some additional feedback: 

- Under the heading of "Health & Safety", it reads "The University of Wisconsin has collected data on various health and safety factors, and created a ranking for each."  Now that a lot of the data is sourced from Dartmouth College, I think this sub-heading should be updated or made more generic. This general feedback also applies to all of the other main categories: Economy, Demographics, Education, and Housing & Living

**Comment by davelandry** [comment]
Thanks for the feedback @anissa-talantikite and @rexley. In the future, please create separate Github Issues for each topic/idea. Here are my notes on each bullet:
* In Data USA terminology, a "course" is a higher education concentration, and the classification of courses used by our data ([CIP](https://nces.ed.gov/ipeds/cipcode/Default.aspx?y=55)) calls the different nesting depths 2-digit, 4-digit, and 6-digit. For example:  Visual & Performing Arts (2-digit) > Design & Applied Arts (4-digit) > Graphic Design (6-digit). This level of nesting is similar to how we say "County" or "State" for a location. In general, we decided to stick as close as possible to the nomenclature of the underlying data, but are open to suggestions to help increase clarity.
* Moved to #488
* Moved to #489
* I'm unable to follow your steps to reproduce this bug. When going to the Cart, how were you able to click on a county?
* Our demographic data does not take citizenship into account, so the "Asians" in this case are both "Asian-Americans" and foreign-born aliens living in the US.
* Moved to #490

And @rexley:
* Moved to #491

**Comment by anissa-talantikite** [comment]
Got it, will do. For the bug relating to the Cart I’m also not able to reproduce it, it must be fixed. I took a screenshot of it before – here it is:


  *   When I went to Cart > clicked on a county > Options, I wasn’t able to choose from the 4 options (Share, Download, Data, API). Just showed a blank screen – see below:
[cid:image003.jpg@01D2AEC2.82CC20F0]

Congrats on all the progress, it really looks great ☺

Best,
Anissa
